code/datums/components/crafting/recipes/misc.dm

```diff
@@ -137,15 +137,6 @@
 	category = CAT_MISC
 	tools = list(TOOL_WELDER)
 
-/datum/crafting_recipe/ipickaxe
-	name = "Improvised Pickaxe"
-	reqs = list(
-			/obj/item/crowbar = 1,
-			/obj/item/kitchen/knife = 1,
-			/obj/item/stack/tape = 1)
-	result = /obj/item/pickaxe/improvised
-	category = CAT_MISC
-
```

Comment:
Why remove this?

Reply:
One, because it's in the lathe now. Two, apogee said to. Three, I've been told it's not used either way.

Reply:
Stuff changed, no longer related.